### ISSUE TITLE
Doesn't seem to work with Go

### ISSUE THREAD

**Issue by aaronkavlie-wf** [atom/language-todo#31]
For some reason this isn't highlighting with Go syntax highlighting for me.

Running Atom 1.0.3, go-plus, go-oracle, etc. (not sure if it's impacted by another package).

**Comment by kevinsawicki** [comment]
Mind including a quick code snippet that exhibits the issue?

**Comment by aaronkavlie-wf** [comment]
Sure:

```go
package main

import "fmt"

func main() {
	// TODO: get this to highlight properly
	fmt.Println("Hello world!")
}
```

**Comment by kevinsawicki** [comment]
Thanks for reporting this, it will be fixed in the next Atom release, 1.0.4

**Comment by aaronkavlie-wf** [comment]
awesome, thanks!

:+1:
